src/tqec/compile/compile.py

```diff
@@ -46,7 +47,7 @@ def __post_init__(self) -> None:
         # Use warning instead of exception because a qec circuit without observable
         # may still be useful, e.g. do statistics on the detection events.
         if len(self.observables) == 0:
-            raise TQECWarning("The compiled graph includes no observable.")
+            warnings.warn("The compiled graph includes no observable.")
```

Comment:
```suggestion
            warnings.warn("The compiled graph includes no observable.", TQECWarning)
```
this helps the user filter warnings if needed.

Reply:
Note that the import also need to be included back.